frontend/src/screens/apps/Index.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable react-refresh/only-export-components */
+import { useEffect, useState } from "react";
+import { useNavigate } from 'react-router-dom';
+
+import Loading from '../../components/Loading';
+import { App, NostrEvent, ListAppsResponse } from "../../types";
```

Comment:
It would be nice to add typescript path aliases from the beginning instead of relative imports